do_with_in_base/src/lib.rs

```diff
@@ -69,10 +70,14 @@ impl Default for Sigil {
   }
 }
 
+/// Escape sequence style set in [Configuration].
 #[derive(Debug,Copy,Clone,PartialEq,Eq)]
 pub enum Escaping {
+  /// No escape sequence is allowed.
```

Comment:
It isn't that escape sequences are not allowed, it is that they don't do anything special - so eg if you have Dollar as the sigil, and `$foo` has value ⌜bar⌝, then ⌜\$foo⌝ will be processed into ⌜\bar⌝, and ⌜$$foo⌝ will be processed into ⌜$bar⌝. Though it is also worth noting that at the moment, stray ⌜\⌝ tokens explode; more work is needed in various places to make backslash escaping work.

Reply:
Github text entry field eating my comments, bleh. \\$foo will be processed into \bar

Reply:
$foo will be processed into bar